package.json

```diff
@@ -16,7 +16,7 @@
     "commit": "git-cz",
     "coverage:collect": "ts-node ./scripts/collect-test-coverage.ts",
     "coverage": "codecov || exit 0",
-    "dev-build": "yarn --cache-folder ~/.cache/yarn && nx run-many --target=build --all",
+    "dev-build": "yarn --network-concurrency 1 --cache-folder ~/.cache/yarn && nx run-many --target=build --all",
```

Comment:
@sobolk Am I crazy? I thought you merged in something a few days ago that did this?

Reply:
see comment below.